planning.py

```diff
@@ -191,22 +221,22 @@ def air_cargo():
     #  Load
     precond = [expr('At(c, a)'), expr('At(p, a)'), expr('Cargo(c)'), expr('Plane(p)'), expr('Airport(a)')]
     effect = [expr('In(c, p)'), expr('~At(c, a)')]
-    load = Action(expr('Load(c, p, a)'), precond, effect)
+    load = PlanningAction(expr('Load(c, p, a)'), precond, effect)
```

Comment:
I think it would be better if `PlanningAction` accepted strings or Exprs, and then this could be

```
load = PlanningAction('Load(c, p, a)',
         'At(c, a), At(p, a), Cargo(c), Plane(p), Airport(a)', 
         'In(c, p), ~At(c, a)')
```

and similarly elsewhere.